src/public/js_native_ext_api.h

```diff
@@ -92,8 +97,8 @@ typedef struct napi_ext_env_settings {
   // Size of this struct to allow extending it in future.
   size_t this_size;
 
-  // Custom scheduler of the foreground JavaScript tasks.
-  napi_ext_schedule_task_callback foreground_scheduler;
+  // Task runner for the foreground JavaScript tasks.
+  v8_task_runner_t foreground_task_runner;
```

Comment:
>foreground_task_runner [](http://example.com/codeflow?start=19&length=22)

I'm curious to see the RNW changes which creates this .. Specifically will this take a reference to the runtime holder or the JS queue ?

Reply:
This is the RNW PR: https://github.com/microsoft/react-native-windows/pull/10966
Yes, it holds a reference to the JS queue.